### PR TITLE
[CircleGraph] Handle loadmodel for multi packet and error

### PR DESCRIPTION
**Pull request by seanshpark**
This will implement loadmodel for multi packet and error handling.

ONE-vscode-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
from draft #482

File: src/CircleGraph/CircleGraph.ts [hunk 1]
```diff
@@ -245,13 +245,21 @@ export class CircleGraphPanel {
...
+        this.handleLoadError(err);
       }
     } else {
-      // TODO impelment multi packet
+      var nextPacketSize = this._modelLength - offset;
+      if (nextPacketSize > sendPacketSize) {
+        nextPacketSize = sendPacketSize;
+      }
+      try {
+        this.sendModelMulti(nextPacketSize, offset);
```
Comment (llFreetimell) [inline src/CircleGraph/CircleGraph.ts:259]: FYI, https://google.github.io/styleguide/jsguide.html#features-use-const-and-let says that "The var keyword must not be used." :)
(I learned this from https://github.com/Samsung/ONE-vscode/pull/333#discussion_r824482027)


```suggestion
      try {
        const nextPacketSize = Math.min(this._modelLength - offset, sendPacketSize);
        this.sendModelMulti(nextPacketSize, offset);
```

Comment (seanshpark) [reply src/CircleGraph/CircleGraph.ts:259]: Your suggestion doesn cover when `this._modelLength - offset > sendPacketSize` so cannot apply this. I'll use `let`

Comment (llFreetimell) [reply src/CircleGraph/CircleGraph.ts]: I think it covers..?

- `Math.min(this._modelLength - offset, sendPacketSize)`
  - returns `sendPacketSize`, when `this._modelLength - offset > sendPacketSize`
  - returns `this._modelLength - offset`, when `this._modelLength - offset <= sendPacketSize`

Comment (seanshpark) [reply src/CircleGraph/CircleGraph.ts]: Ah, I didn't notice `Math.min();` ! thanks!

**Review by llFreetimell — APPROVED** [review]

**Pull request merged**